src/Protocol/Protocol_1_8.h

```diff
@@ -176,6 +173,9 @@ class cProtocol_1_8_0 :
 	/** Adds the received (unencrypted) data to m_ReceivedData, parses complete packets */
 	void AddReceivedData(const char * a_Data, size_t a_Size);
 
+	/** Nobody inherits 1.8, so it doesn't use this method */
+	virtual UInt32 GetPacketId(eOutgoingPackets a_Packet) override { ASSERT(!"GetPacketId for cProtocol_1_8_0 is not implemented."); return 0; }
```

Comment:
This deserves a properly indented function body.

Reply:
Indented.